Dockerfile

```diff
@@ -4,7 +4,7 @@
 ARG build_dir=/build_application
 
 # first stage which builds the application
-FROM node:14
+FROM node:18
```

Comment:
@matthiaslehnertum may I ask about the reasoning for this? global-jsdom [requires node 16](https://github.com/modosc/global-jsdom/blob/9e9e85b07c2a8135496ad7bf5614f22ddd2f63cc/packages/global-jsdom/package.json#L55), and the rest of the build was being done with node 14 previously.